packages/apps-config/src/endpoints/util.ts

```diff
@@ -52,9 +54,12 @@ export function expandEndpoint (t: TFunction, { dnslink, genesisHash, homepage,
     .map(([host, value], index): LinkOption => ({
       ...base,
       dnslink: index === 0 ? dnslink : undefined,
+      isLightClient: typeof value === 'string' ? false : value.type === 'substrate-connect' as EndpointType,
       isRelay: false,
-      textBy: t('rpc.hosted.by', 'via {{host}}', { ns: 'apps-config', replace: { host } }),
-      value
+      textBy: typeof value === 'string'
```

Comment:
Prefer not using `typeof value`, `isString(value)` would do the job. (No difference, just a helper, everywhere where used here)